spec/cspec/RISCV64/Kernel_C.thy

```diff
@@ -26,7 +26,7 @@ declare [[populate_globals=true]]
 context begin interpretation Arch . (*FIXME: arch_split*)
 
 type_synonym cghost_state = "(machine_word \<rightharpoonup> vmpage_size) * (machine_word \<rightharpoonup> nat)
-    * ghost_assertions"
+    * ghost_assertions * (machine_word \<rightharpoonup> nat)"
```

Comment:
While you're here it would be nice to fix up this indentation. If it fits it would just be moving the whole type to the second line. If not, I guess something like this?
```
type_synonym cghost_state =
  "(machine_word ⇀ vmpage_size) * (machine_word ⇀ nat) * ghost_assertions *
   (machine_word ⇀ nat)"
```

Reply:
Done!